src/pluginload.c

```diff
@@ -304,7 +304,7 @@ plugins_reload_from_dir (void *folder)
         g_message ("Loading %s", name);
       if (g_str_has_suffix (name, ".nasl"))
         {
-          if (nasl_plugin_add (folder, name))
+          if (nasl_plugin_add ((char *) folder, name))
```

Comment:
Why is the const qualifier being removed? wouldn't be better to make const also the parameter in the nasl_plugin_add() (in src/nasl_plugins.c) ?

Reply:
yes.